front/src/components/Task/TaskCard.vue

```diff
@@ -20,7 +20,9 @@
     </q-card-section>
     <q-separator v-show="!isUltraDense" inset />
     <q-card-section v-if="!isDense">
-      <task-player-list :task="task" style="min-height: 36px" />
+      <div class="row" style="min-height: 36px">
+        <task-player-list :task="task" />
+      </div>
```

Comment:
Why this?

Reply:
I thought that change alongside this one made things a bit compacter, but after testing again it doesn't really matter. Will revert it.